apps/server/cypress/e2e/suite1/search.cy.ts

```diff
@@ -97,7 +97,7 @@ describe("search", () => {
 
   describe("Search by routing number", () => {
     it('shows "America First Credit Union" when the routing number is entered into search', () => {
-      visitAgg({});
```

Comment:
I made the params optional in the `visitAgg` function so we don't have to pass an empty object.